bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -406,14 +407,20 @@ export default {
         this.$f7.preloader.hide()
         this.$f7.dialog.alert('Error while signing out: ' + err)
       })
+    },
+    updateThemeOptions () {
+      this.themeOptions.dark = localStorage.getItem('openhab.ui:theme.dark') || (this.$f7.darkTheme ? 'dark' : 'light')
+      this.themeOptions.bars = localStorage.getItem('openhab.ui:theme.bars') || ((!this.$theme.ios) || (!this.$f7.darkTheme) ? 'filled' : 'light')
+      this.themeOptions.homeNavbar = localStorage.getItem('openhab.ui:theme.home.navbar') || 'default'
+      this.themeOptions.expandableCardAnimation = localStorage.getItem('openhab.ui:theme.home.cardanimation') || 'default'
+      this.themeOptions.pageTransitionAnimation = localStorage.getItem('openhab.ui:theme.home.pagetransition') || 'default'
+      // f7 adds theme-dark to the class list if autoDarkTheme is enabled
+      if (this.$f7.darkTheme && this.themeOptions.dark === 'light') {
+        document.querySelector('html').classList.remove('theme-dark')
```

Comment:
in order to use the built in dark detection,  f7 is going to inject the dark-theme into the class list, not sure if there's a better way of removing it using the framework (or css for that matter) See the code in question for more info:
https://github.com/framework7io/framework7/blob/080ea7e7cea5c3332ad211f263f1ac61dd37dfff/src/core/components/app/app-class.js#L107  ,

Reply:
Maybe you could set autoDarkTheme to true when initializing the app only if `localStorage.getItem('openhab.ui:theme.dark')` returns nothing? Otherwise it means the user has overridden the theme.

Reply:
that's a very good idea